cdap-ui/cdap-ui-installer/index.js

```diff
@@ -0,0 +1,137 @@
+var yauzl = require("yauzl");
+var path = require("path");
+var mkdirp = require("mkdirp");
+
+var fs = require('fs');
+var archiver = require('archiver');
+var minimist = require('minimist');
+var Listr = require('listr');
+
+var runtimearguments = minimist(process.argv.slice(2));
+var newuizippath = runtimearguments['new-ui-zip-path'];
+var cdaphome = process.env.CDAP_HOME;
+
+var folderstozip = [
+  '/dist/',
+  '/cdap_dist/',
+  '/node_modules/',
+  '/common_dist/',
+  '/dll/',
+  '/login_dist/',
+  '/old_dist/',
+  '/server/',
+  '/templates/',
+  '/wrangler_dist/'
+];
+var filestozip = [
+  '/package.json',
+  '/server.js/',
+  '/dist/VERSION'
```

Comment:
We don't have README in a SDK build nor in a cluster right? I am just archiving existing files from sdk/cluster ui